packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js

```diff
@@ -118,6 +119,23 @@ describe('ListToVirtualizedList', () => {
 		});
 	});
 
+	it('should support custom cell renderer', () => {
+		const clonedProps = cloneDeep(props);
+		const renderer = function test() {
+			return 'ok';
+		};
+		clonedProps.cellDictionary = { customType: { cellRenderer: renderer } };
+		const wrapper = shallow(<ListToVirtualizedList {...clonedProps} />);
+
+		// then
+		wrapper.find(VirtualizedList.Content).forEach(element => {
```

Comment:
Insteed of going through all the columns, `Array.find()` the wanted column and test the assertion on it